environment.yml

```diff
@@ -17,4 +17,4 @@ dependencies:
     - deepdish
     - SimPEG
     - empymod
-    - git+https://github.com/geoscixyz/geosci-labs
+    - git+https://github.com/geoscixyz/geosci-labs.git@tomography-inversion
```

Comment:
@sgkang: would you mind reverting this? when we merge we will delete the branch, and this would break the binder